### PR TITLE
Basic auth

### PR DESCRIPTION
**Pull request by niallryan**
@maigfrga

### CODE REVIEW

**Comment by mfranco** [comment]
@cormacp @felipepina01

**Comment by mfranco** [comment]
:+1:

File: test/tests/config/ci.py [hunk 1]
```diff
@@ -1,3 +1,7 @@
 from .development import *  # noqa
```
Comment (cormacp) [inline test/tests/config/ci.py:1]: Could be worth checking if @Rondineli needs to re-run his Chef config scripts for these config files

Comment (niallryan) [reply test/tests/config/ci.py:1]: This is just a test app so should be fine.

**Pull request merged**